server/utils/passport/ensureStudyPermission.js

```diff
@@ -0,0 +1,22 @@
+import UserModel from '../../models/UserModel'
+
+export default async function ensureStudyPermission(req, res, next) {
+  if (!req.isAuthenticated())
+    return res.status(401).json({ error: 'Unathorized' })
+  try {
+    const { study } = req.params
+    const { appDb } = req.app.locals
+    const user = await UserModel.findOne(appDb, { uid: req.user })
+    const { blocked, access } = user
+    const isStudyInAccess = user.access.includes(study)
+
+    if (!access.length || !isStudyInAccess)
```

Comment:
is it possible for a user to have an empty `access` and have `isStudyInAccess` be true? I don't _think_ so, but want to check. I believe we can change this to only check `isStudyInAccess`.

Reply:
[9ac5eed](https://github.com/AMP-SCZ/dpdash/pull/344/commits/9ac5eed8b55530c29fa5dc110bfc7de660805119)